README.md

```diff
@@ -87,6 +87,30 @@ days thence that Event will be processed.
 - Routine scheduling behaviors allow rules like run-once, schedule-once, last-only (cancel any existing Events when one is scheduled)
 - Task groups allow scheduling behavior to be applied to multiple Routines within a group
 
+# Config
+`UMEBOSHI_SERIALIZER`: Path to custom data serializer 
+
+Example:
+```
+# In Django setting file
+
+UMEBOSHI_SERIALIZER = "example.path.serializer.CustomSerializer"
```

Comment:
@markhsu-ms What is the rationale for supporting CustomSerializer?

Reply:
In our use case is to make Pickling compatible between Py2 and Py3. See this [PR](https://github.com/makingspace/site/pull/11344)

Django-Redis has similar approach ([link](https://django-redis-cache.readthedocs.io/en/latest/advanced_configuration.html#pluggable-serializers))

Reply:
Understood.